code/surface_area/compute_surface_area.py

```diff
@@ -0,0 +1,172 @@
+import argparse
+from pathlib import Path
+from typing import Iterable
+
+from niwrap import workbench as wb
+from styxdocker import DockerRunner
```

Comment:
Nice - more a note than a change, for quick scripts, you can also use `niwrap.use_docker()` and `get_global_runner` to grab the runner. Just avoids having to import all the styx libraries for different libraries if we want to be able to specify the runner.